packages/dbos-cloud/userdb.ts

```diff
@@ -10,15 +10,15 @@ export interface UserDBInstance {
   readonly AdminUsername: string;
 }
 
-export async function createUserDb(host: string, dbName: string, adminName: string, adminPassword: string, sync: boolean) {
+export async function createUserDb(host: string, dbName: string, appDBUsername: string, appDBPassword: string, sync: boolean) {
   const logger = getLogger();
   const userCredentials = getCloudCredentials();
   const bearerToken = "Bearer " + userCredentials.token;
 
   try {
     await axios.post(
       `https://${host}/v1alpha1/${userCredentials.userName}/databases/userdb`,
-      { Name: dbName, AdminName: adminName, AdminPassword: adminPassword },
+      { Name: dbName, AdminName: appDBUsername, AdminPassword: appDBPassword },
```

Comment:
Shouldn't the field names be `Name` and `Password`? (And we would update the API struct flags accordingly?)

Reply:
That belongs to another PR -- in some sense this is also an admin role because they can view their entire database.